### ISSUE TITLE
IMU preintegration actually not used?

### ISSUE THREAD

**Issue by yinhaosen** [Gaochao-hit/LIO-SAM-extending1#8]
https://github.com/Gaochao-hit/LIO-SAM_based_relocalization/blob/b162b829660d34ab5968b47d4d1a12f23f54c0f1/src/globalLocalize.cpp#L463

Here, in`globalLocalize.cpp`, it seems that imu preintegration odometry for prior to lidar actually cannot be used because the code of lidar incremental pose has been deleted.

**Comment by NozomiWyane** [comment]
I want to ask the same question, it seems that it didnt use the imupreintegration results in code, but I try to use it in the code. Have you tried it?

**Comment by yinhaosen** [comment]
> I want to ask the same question, it seems that it didnt use the imupreintegration results in code, but I try to use it in the code. Have you tried it?

Yes. Just same as the original version of LIO-SAM but you need to do some tricks to integrate them. I have completed it a few months ago and forgotten what tricks :(
